packages/core/components/package.json

```diff
@@ -14,7 +14,7 @@
     "access": "public"
   },
   "devDependencies": {
-    "parcel": "^2.8.2",
+    "parcel": "2.8.2",
```

Comment:
Why we want to stick on this version

Reply:
I'm reverting this, something was apparently causing the build issues in `2.8.3` but I can't find the github issue anymore.

Also, the latest one seems to be working correctly now.